### ISSUE TITLE
sftp support!?

### ISSUE THREAD

**Issue by Borsti26** [trippo/ResponsiveFilemanager#545]
Hello,

is it possible to use sftp connection?
If yes, how?

BR

**Comment by markmorris** [comment]
Yeah, been looking for this, im not sure if anyone still using FTP because of how insecure it is. SFTP would be so much better.
